src/project-tests/pomodoro-clock-tests.js

```diff
@@ -1,13 +1,13 @@
-/*globals FCC_Global */
+import { assert } from 'chai';
 
 export default function createPomodoroClockTests() {
 
-  const _break_min = "break-decrement";
-  const _break_plus = "break-increment";
-  const _sesh_min = "session-decrement";
-  const _sesh_plus = "session-increment";
-  const _reset = "reset";
-  const _start_stop = "start_stop";
+  const _breakMin = 'break-decrement';
```

Comment:
I'm on the fence about whether or not to rename variables during an "ESLint only" PR, but I think we should.... it's a minor change we don't want to mix in with more significant changes. These should all have the leading underscore removed.